api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -579,7 +588,7 @@ def touch_tip(
             return self
         if parent_labware.is_tiprack:
             _log.warning(
-                "Touch_tip being performed on a tiprack. Please re-check your code"
+                "Touch_tip being performed on a tip rack. Please re-check your code"
```

Comment:
Not a docstring, could affect tests. Revert.
```suggestion
                "Touch_tip being performed on a tiprack. Please re-check your code"
```